tcl/tools/reviewgame.tcl

```diff
@@ -368,7 +372,11 @@ proc ::reviewgame::checkPlayerMove {} {
     # after 1000 ::reviewgame::mainLoop
     # return
   }
-  
+  if { $moveForward } {
+      sc_var exit
+      sc_move forward
+      updateBoard -pgn -animate
```

Comment:
updateBoard is the old function and should be used only to just update the board.
New code should use ::notify::Poschanged and it is probably not necessary to redraw the pgn.
Even better would be if it is possible to use ::move::Forward